pennylane/optimize/qng.py

```diff
@@ -146,6 +146,11 @@ class QNGOptimizer(GradientDescentOptimizer):
         See the `quantum natural gradient example <demo:demos/tutorial_quantum_natural_gradient>`_
         for more details on the Fubini-Study metric tensor and this optimization class.
 
+    .. seealso::
+
+        See the :class:`~.QNGOptimizerQJIT` for an Optax-like and ``jax.jit``/``qml.qjit``-compatible
+        implementation of the same Quantum Natural Optimizer algorithm.
```

Comment:
Oh great catch! @SimoneGasperini maybe we can add this into the v0.42 release? We can re-target this PR to the v0.42.0-docs branch :). cc @andrijapau